client/ayon_core/tools/utils/views.py

```diff
@@ -19,48 +18,6 @@ def mousePressEvent(self, event):
         QtWidgets.QTreeView.mousePressEvent(self, event)
 
 
-class TreeViewSpinner(QtWidgets.QTreeView):
```

Comment:
Can we just comment it out?

Reply:
We could - but seems messy? :) Or did you mean to then remove it in a separate PR?

Reply:
It is messy, but at least some reminder that we actually did have refresh animations...

Reply:
I think we're better off maybe creating an issue if we want to keep track of that - it could e.g. link to the removal commit. What you think?

Reply:
Also an option, yes.